README.md

```diff
@@ -0,0 +1,101 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- java sdk
+description: "Sample code repo for Azure Cosmos DB Java SDK for SQL API"
+urlFragment: ""
+---
+
+# Azure Cosmos DB Java SQL API Samples
+
+<!-- 
+Guidelines on README format: https://review.docs.microsoft.com/help/onboard/admin/samples/concepts/readme-template?branch=master
+
+Guidance on onboarding samples to docs.microsoft.com/samples: https://review.docs.microsoft.com/help/onboard/admin/samples/process/onboarding?branch=master
+
+Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
+-->
+
+Sample code repo for Azure Cosmos DB Java SDK for SQL API. By cloning and running these samples, and then studying their implementations, you will have an example for sending various requests to Azure Cosmos DB from Java SDK via the SQL API.
+
+## Contents
+
+| File/folder       | Description                                |
+|-------------------|--------------------------------------------|
+| `src`             | Java sample source code. Many samples have 'sync' and 'async' variants                |
+| `.gitignore`      | Define what to ignore at commit time.      |
+| `CHANGELOG.md`    | List of changes to the sample.             |
+| `CONTRIBUTING.md` | Guidelines for contributing to the sample. |
+| `README.md`       | This README file.                          |
+| `LICENSE`         | The license for the sample.                |
+| `pom.xml`         | Maven Project Object Model File
+
+## Prerequisites
+
+* A Java IDE such as IntelliJ IDEA or VSCode
+* Maven
+* Java SE JRE 8
+* Setting up an Azure Cosmos DB account through the Azure Portal. The **Create a database account** section of [this guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-sql-api-java) walks you through account creation. 
+* The hostname and master key for your Azure Cosmos DB account
+
+## Setup
+
+Clone the sample to your PC. Using your Java IDE, open pom.xml as a Maven project.
+
+## Running the sample
+
+*If you are using Intellij IDEA as your Java IDE:* Once you have opened the project, go to the **Run/Debug Configurations** drop-down and choose **Edit Configurations**. 
```

Comment:
Some people use Eclipse as the IDE of choice; I will recommend removing this reference because people might think that Microsoft prefers IntelliJ over other tools.

instead just keep a simple note that these environment variables must be set before the samples can be executed.